tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -76,7 +76,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
     {
         Logger::Warn("DATADOG TRACER DIAGNOSTICS - Profiler disabled: .NET 5.0 runtime or greater is required on this "
                      "architecture.");
-        return E_FAIL;
+        return CORPROF_E_PROFILER_CANCEL_ACTIVATION;
```

Comment:
I think this one could still be a "fail". Users tried to load the tracer, but we don't support the current environment.

Reply:
+1, we'd want to raise an error in the event log for this scenario